core/drivers/stm32_rtc.c

```diff
@@ -358,6 +426,20 @@ static TEE_Result parse_dt(const void *fdt, int node)
 	if (res)
 		return res;
 
+	if (IS_ENABLED(CFG_RTC_PTA) && IS_ENABLED(CFG_DRIVERS_RTC)) {
```

Comment:
Why is this dependent on PTA presence? We should parse the property irrelevant of everything and create a handler later on the RTC secure interrupt if at least one resource is managed by OP-TEE IMO.